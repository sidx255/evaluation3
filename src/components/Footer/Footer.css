.footer {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    background-color: #121212;
    padding-top: 2rem;
    padding-bottom: 2rem;
    color: white;
    bottom: 0;
    /* width: 100%; */
    padding-left: 30px;
}

.themes {
    display: flex;
    flex-direction: row;
    gap: 1rem;
}
  