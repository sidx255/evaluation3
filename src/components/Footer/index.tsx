import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className='body-padding footer'>
      <div className='footer-text'>THEMES</div>
      <div className="themes">
        <div className="theme-color" style={{ backgroundColor: '#000000' }}>
            blue
        </div>
        <div className="theme-color" style={{ backgroundColor: '#1f1f1f' }}>
            green
        </div>
        <div className="theme-color" style={{ backgroundColor: '#2f2f2f' }}>
            red
        </div>
      </div>
    </div>

    
  );
};

export default Footer;