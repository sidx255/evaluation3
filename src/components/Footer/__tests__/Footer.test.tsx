import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../index';

test('renders app', () => {
  render(<Footer />);
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});