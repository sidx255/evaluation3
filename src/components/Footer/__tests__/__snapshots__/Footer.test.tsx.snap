// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<DocumentFragment>
  <div
    class="body-padding footer"
  >
    <div
      class="footer-text"
    >
      THEMES
    </div>
    <div
      class="themes"
    >
      <div
        class="theme-color"
        style="background-color: rgb(0, 0, 0);"
      >
        blue
      </div>
      <div
        class="theme-color"
        style="background-color: rgb(31, 31, 31);"
      >
        green
      </div>
      <div
        class="theme-color"
        style="background-color: rgb(47, 47, 47);"
      >
        red
      </div>
    </div>
  </div>
</DocumentFragment>
`;
